ome_model/experimental.py

```diff
@@ -46,6 +46,26 @@ def __init__(self,
         Channel.ID += 1
 
 
+class Plane(object):
+
+    ALLOWED_KEYS = (
+        'DeltaT', 'DeltaTUnit', 'ExposureTime', 'ExposureTimeUnit',
+        'PositionX', 'PositionXUnit', 'PositionY', 'PositionYUnit',
+        'PositionZ', 'PositionZUnit')
+
+    def __init__(self, TheC=0, TheZ=0, TheT=0, options={}):
+        self.data = {
+            'TheC': str(TheC),
+            'TheZ': str(TheZ),
+            'TheT': str(TheT),
+        }
+        if not options:
```

Comment:
nitpick: I'd tend to do `if options: ...` since a return in a constructor can be a surprise for someone who adds later logic.

Reply:
Fixed in a2819f8